puzzles/p2_delegated_singleton_message.clsp

```diff
@@ -0,0 +1,31 @@
+; This puzzle is used to delegate a spend to a singleton.
+; The singleton will authorize the spend by sending a message via the SEND_MESSAGE condition.
+(mod (
+    SINGLETON_MOD_HASH
+    SINGLETON_STRUCT_HASH ; The hash of (SINGLETON_MOD_HASH . (LAUNCHER_ID . SINGLETON_LAUNCHER_HASH))
+    NONCE
```

Comment:
Since this is a brand-new puzzle, we can use the MIPS for it, no?  Since we're switching to MIPS, we want to minimize the non-MIPS chialisp out there.

Reply:
Yeah since we settled on using MIPS, removed this from the PR